add-collaborators.rb

```diff
@@ -0,0 +1,65 @@
+# add-collaborators.rb
+# add as collaborators anyone who comments on a given issue in a given repo
+# assumes you created and stored the appropriate OAuth token as an ENV variable called GITHUBTEACHER_TOKEN
+
+require 'octokit'
+require 'optparse'
+
+
+# Make sure arguments are specified
+ARGV << '-h' if ARGV.empty?
+
+
+# Create options hash
+options = {}
+
+
+# Parse options
+OptionParser.new do |opts|
+  opts.banner = "Usage: add-collaborators.rb [options]"
+
+  opts.on("-r", "--repo REPOSITORY", "Repository Path -- ex: githubteacher/example-repo") do |r|
+    options[:repo] = r
+  end
+
+  opts.on("-i", "--issue ISSUE", Numeric, "Issue Number -- ex: 4") do |i|
+    options[:issue] = i
+  end
+
+  opts.on_tail("-h", "--help", "Prints this help message") do |h|
+    puts opts
+    exit
+  end
+
+end.parse!
+
+puts options
+
+
+# Assign variables
+token = 'GITHUBTEACHER_TOKEN'
```

Comment:
No, wait. I get it, I'm just really dumb.

In case I'm not the only one: this isn't the token, it's the key we're using to get the token from ENV. As in, we expect the ENV to have `GITHUBTEACHER_TOKEN="abc123"`.

I still think there might be a more "ruby" way to do this, but this line is fine as is. I am just being slow today. (Lately?)